iam-login-service/src/test/java/it/infn/mw/iam/test/api/tokens/AccessTokenGetRevokeTests.java

```diff
@@ -106,7 +106,7 @@ public void getAccessToken() throws JsonParseException, JsonMappingException,
     System.out.println(remoteAt);
 
     assertThat(remoteAt.getId(), equalTo(at.getId()));
-    assertThat(remoteAt.getValue(), equalTo(at.getValue()));
+    assertThat(remoteAt.getValue(), equalTo(null)); 
```

Comment:
There's a better matcher to check for null values, IIRC is called  `nullValue()`

Reply:
Fixed, 
I have a question about the usage of this matcher regarding the code efficiency.. for my education purpose only... we talk when you're back :)